plane/src/drone/command.rs

```diff
@@ -10,7 +10,7 @@ use clap::Parser;
 use std::{net::IpAddr, path::PathBuf};
 use url::Url;
 
```

Comment:
nit: rm newline

Reply:
sorry about this happening a lot, VSCode seems to do this automatically -- have you found a way to turn this off or are you just diligent about it?